src/internal/connector/graph_connector.go

```diff
@@ -349,30 +305,27 @@ func getResources(
 
 	response, err := query(ctx, gs)
 	if err != nil {
-		return nil, errors.Wrapf(
-			err,
-			"retrieving resources for tenant %s: %s",
-			tenantID,
-			support.ConnectorStackErrorTrace(err),
-		)
+		return nil, clues.Wrap(err, "retrieving tenant's resources").
+			WithClues(ctx).
+			WithAll(graph.ErrData(err)...)
 	}
 
+	errs := &multierror.Error{}
```

Comment:
I take it there'll be another pass to switch this to the fault package?

Reply:
Yup, just didn't want to overload this PR.